Site/SiteMediaDurationUpdater.php

```diff
@@ -0,0 +1,277 @@
+<?php
+
+require_once 'Site/SiteCommandLineApplication.php';
+require_once 'Site/SiteCommandLineArgument.php';
+require_once 'Site/SiteConfigModule.php';
+require_once 'Site/SiteDatabaseModule.php';
+require_once 'Site/dataobjects/SiteMediaWrapper.php';
+require_once 'Site/dataobjects/SiteAudioMedia.php';
+
+/**
+ * Recalculates the duration of all media in a media set
+ *
+ * @package   Site
+ * @copyright 2015 silverorange
+ */
+class SiteMediaDurationUpdater extends SiteCommandLineApplication
```

Comment:
Should this be `SiteAudioMediaDurationUpdater`? It only works with `SiteAudioMedia`. Alternatively, `parseDuration()` should move to `SiteMedia`.